src/main/java/com/spotify/docker/client/messages/HostConfig.java

```diff
@@ -969,6 +969,10 @@ public static BuilderFrom from(final String from) {
       return new BuilderFrom(from);
     }
 
+    public static BuilderFrom from(final Volume volumeFrom) {
```

Comment:
I see thanks for explaining. In that case why only add this method and not `BuilderTo to(final Volume volumeTo)`?